model/Scenario.py

```diff
@@ -0,0 +1,60 @@
+from abc import ABC
+from model import Gadgets
+
+
+class FieldState(ABC):
```

Comment:
use enum ?

Reply:
Erst mal nicht, würde gerne Später evtl. das is destroyed und ähnliches direkt in den FieldState mit reinnehmen. Muss dafür aber nochmal im Lastenheft lesen, was das alles sein kann usw.